### DISCUSSION TITLE
I can't find out how is do to get categories of products on ETSY API

### DISCUSSION THREAD

**Discussion by FatmaSedaOZYURT** [etsy/open-api#752] [Q&A] [answered]
I want to create category tree. But, I can't do this how can I make a request? 
Is this possible on API?
Thanks in advance.

**Comment by baahar** [comment, marked as answer]
Yes, there is a seller taxonomy: https://developers.etsy.com/documentation/reference/#operation/getSellerTaxonomyNodes

**Comment by FatmaSedaOZYURT** [comment]
When i see taxonomy, I thought it was tax related because of name. 
It is due to my vacuity. Sorry about this.
Thank you @baahar for enlightening me.
